### ISSUE TITLE
Document view mode: vertical scroll

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#474]
```
Hi. I have problem with Document view mode: only vertical scroll.
When i open again book on last reading page and i try scroll to next page 
ebookdroid back to the first page. This is very annoying, please fix this. I 
use Sony Xperia U android 4.0.4 and EBookDroid 1.6.6
```

Original issue reported on code.google.com by `robarob...@gmail.com` on 23 Mar 2013 at 5:21

**Comment by GoogleCodeExporter** [comment]
```
Issue 483 has been merged into this issue.
```

Original comment by `Andrei.K...@gmail.com` on 8 Apr 2013 at 8:25

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.0
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 25 Dec 2013 at 12:00
- Changed state: **Fixed**
